src/lib/axios.test.js

```diff
@@ -87,4 +87,43 @@ describe("axios", () => {
         .undefined;
     });
   });
+
```

Comment:
Is there a specific test that tests we are not sending an `x-forwarded-for` header of `null` ? 

https://github.com/alphagov/di-ipv-cri-common-express/pull/89#discussion_r995923915